### PR TITLE
VACMS-4096: Configure friendly 403 ErrorDocument for missing WEB builds.

### PR DESCRIPTION
**Pull request by ElijahLynn**
## Description

See #4095. 

## Screenshots

### CODE REVIEW

**Comment by ElijahLynn** [comment]
Init stage doesn't run on PRs by default. Manually building this branch with no base preview so that init stage will run.

**Comment by ElijahLynn** [comment]
This is the default message showing now on a new PR branch I just built. 
image

File: .web/403-error-document.html [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+  </style>
+</head>
+
+<body>
+<div class="container">
+  <div class="title-container">
+    <h1 class="page-title">Front End WEB Preview In Progress</h1>
+    <div id="contextual-help">
+      <p class="contextual-help">
+        Sorry, there is no Front End WEB preview available yet. The build either hasn't started yet, or is in progress. This page will refresh every 2 seconds and when the Front End WEB preview is ready it will appear here and this message will go away.
```
Comment (oksana-c) [inline .web/403-error-document.html:434]: @rachel-kauff will you be able to give this text a once over sometime early today?

**Review by indytechcook — APPROVED** [review]

**Comment by ElijahLynn** [comment]
>  Test Failed: `va/tests/phpunit`

This appears unrelated, might be another issue going on.

**Comment by ElijahLynn** [comment]
GIF of newest functionality here https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4095#issuecomment-768857527

**Pull request merged**